.github/CHANGELOG.md

```diff
@@ -34,10 +36,15 @@
   [documentation](https://pennylane.ai/en/stable/code/api/pennylane.beta.plugins.DefaultQubitTF.html)
   for more details.
 
-* Contains the new template `UCCSD` implementing the Unitary Coupled-Cluster (UCCSD) ansatz
-  to perform VQE-based quantum chemistry simulations using PennyLane-QChem.
-  [(#654)](https://github.com/XanaduAI/pennylane/pull/654)
-
+* The ``default.tensor`` plugin has been significantly upgraded. It now allows two different
+  tensor network representations to be used: `"exact"` and `"mps"`. The former uses a 
+  exact factorized representation of quantum states, while the latter uses a matrix product state 
+  representation.
+  ([#572](https://github.com/XanaduAI/pennylane/pull/572))
+  ([#599](https://github.com/XanaduAI/pennylane/pull/599))
```

Comment:
💯

Reply:
Maybe should link to the `default.tensor` docstring, since no example is given here?